docs/source/routing/get-started.mdx

```diff
@@ -0,0 +1,293 @@
+---
+title: Apollo Router Quickstart
+subtitle: Run a router binary
+description: This quickstart tutorial walks you through installing an Apollo Router binary, running it with an example supergraph schema and a YAML configuration file, and making test queries with Apollo Sandbox.
+---
+
+import ElasticNotice from "../../shared/elastic-notice.mdx";
+
+Hello! Let's run Apollo Router for the first time, using the simple scenario of developing and deploying locally.
+
+In this guide, you will:
+
+- Download and install a router binary.
+- Download a supergraph schema.
```

Comment:
```suggestion
- Create a supergraph schema.
```
Even if we start help them make their supergraph schema via download, can we use the word "create" here? I think it's more aligned with how we want the user to think about these things